fastlane.gemspec

```diff
@@ -93,7 +93,7 @@ Gem::Specification.new do |spec|
 
   # The Google API Client gem is *not* API stable between minor versions - hence the specific version locking here.
   # If you upgrade this gem, make sure to upgrade the users of it as well.
-  spec.add_dependency('google-api-client', '>= 0.13.1', '< 0.14.0') # Google API Client to access Play Publishing API
+  spec.add_dependency('google-api-client', '0.21.2') # Google API Client to access Play Publishing API
```

Comment:
Can we use our `>= 0.21.2` `< 1.0` notion? Or `< 0.22` as they don't seem to follow semver

Reply:
@KrauseFx Totes! Can make that a change 👍